ApollosProject/apollos-apps
# feat: Send a push notification when adding a new comment

Testing instructions

1. On your phone (running testflight version of app) login to a user account (user account A) 
2. Locally in Graphql, with this PR running, login to a user in the GraphQL playground (user account B). Make sure you are using the Heroku postgres server.
3. User A follow user B. Doesn't matter how you accomplish this, I added a DB record by hand. 
4. With User B, create a Comment by w/ the GraphQL API. 
5. User A should get a push notification!
